Cargo.toml

```diff
@@ -5,10 +5,11 @@ edition = "2021"
 rust-version = "1.77"
 
 [features]
-default = ["tls-ring"]
+default = ["tls-aws-lc"]
```

Comment:
LGTM, and I think this is a better default anyway (there's nothing very special about `ring` and upstream rust-tls already moved to `aws-lc`), but if we make it default README needs another bump to reflect.

Reply:
Oh I actually meant to make this !default in this PR and split it out